src/OpenTelemetry.Api/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Added a section on Baggage API to README.md.
```

Comment:
not required as we are not making any product change.

Reply:
Fixed this.